examples/observability/eks-cluster-with-amp-amg-opensearch/main.tf

```diff
@@ -165,6 +165,7 @@ resource "grafana_data_source" "prometheus" {
 #---------------------------------------------------------------
 # Provision OpenSearch and Allow Access
 #---------------------------------------------------------------
+#tfsec:ignore:aws-elastic-search-enable-domain-logging
```

Comment:
should be safe to ignore in example - otherwise requires creating CloudWatch log group in example